### DISCUSSION TITLE
Why translations from CW were not imported to 2.1.1?

### DISCUSSION THREAD

**Discussion by thehijacker** [crocodilestick/Calibre-Web-Automated#123] [Q&A]
Hello,

I have put a lot of time into Slovenian translation for CW and was eager to wait for new release of CWA that would include it. But sadly it is not there. Were it not pulled from CW?

Can you please check?

Thank you.

**Comment by crocodilestick** [comment]
Hey @thehijacker , sorry about that! I was hoping they were in the 0.6.23 base CW version we upgraded to with V2.1.0. I don't know how the translation system works but I'll look it up for you on Monday and add them as V2.1.2 asap. Are they already included in a nightly version of CW?

**Reply by thehijacker** [reply]
Yes translation is in CW nightly. I am running it in parallel with CWA in docker just so I could test and improve translation.

**Reply by crocodilestick** [reply]
Ah okay good, I'll have a look Monday and add asap!

**Reply by thehijacker** [reply]
Just for info. I am using this image of CW that has Slovenian translation: lscr.io/linuxserver/calibre-web:nightly

![image](https://github.com/user-attachments/assets/e16132c4-af96-416d-b1db-d842b0c21886)

Maybe a question in advanced. Are there any CWA specific strings that can also be translated?

**Reply by thehijacker** [reply]
> Ah okay good, I'll have a look Monday and add asap!

Still no Slovenian translation. Just updated to latest version. You will wait for CW to release new stable version?

**Reply by crocodilestick** [reply]
Hi @thehijacker , yeah unfortunately for now. I just couldn't really make sense of the translation system and hopefully the next stable CW version isn't too far off 🤞
